src/routing/command.rs

```diff
@@ -71,11 +70,7 @@ pub(crate) enum Command {
         message: MessageType,
     },
     /// Send `UserMessage` with the given source and destination.
-    SendUserMessage {
-        src: SrcLocation,
-        dst: DstLocation,
-        content: Bytes,
-    },
+    SendUserMessage { itry: Itinerary, content: Bytes },
```

Comment:
Minor suggestion which applies to the whole PR, not just this line: I'd spell out `itinerary` in full instead of abbreviating it. It's not very commonly used word and definitely not a commonly used abbreviation, so it just makes the codebase ever so slightly more difficult to read.

Reply:
Yeah, thought of that as well.